.github/workflows/ci.yml

```diff
@@ -67,15 +67,15 @@ jobs:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
       # Only aarch64 needs this, but it doesn't hurt anything
       - name: Install qemu/docker
-        run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        run: docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
Should we just do --install arm64 instead of all?

Reply:
Nevermind, I see that you tried it.

Reply:
yeah, good point. updated